src/bio/tests/bio_ut.c

```diff
@@ -78,6 +78,7 @@ ut_init(struct bio_ut_args *args)
 		D_ERROR("Allocate Per-xstream NVMe context failed. "DF_RC"\n", DP_RC(rc));
 		goto out_smd;
 	}
+	args->bua_seed = (unsigned int)(time(NULL) & 0xFFFFFFFFUL);
```

Comment:
Minor: Per Ashley's comment on the chat, might be good to print the seed in case we need to reproduce a "random" failure

Reply:
Ok, I'll fix it since the patch needs be refreshed anyway (build failed due to jenkins issue).